hw/ip_templates/flash_ctrl/rtl/flash_ctrl_top_specific_pkg.sv.tpl

```diff
@@ -297,7 +297,7 @@ package flash_ctrl_top_specific_pkg;
                  ecc_en:      MuBi4True,
                  he_en:       MuBi4True, // HW assumes high endurance
                  base:        '0,
-                 size:        NumBanks * PagesPerBank
+                 size:        10'(NumBanks * PagesPerBank)
```

Comment:
This commit looks like a candidate for splitting out into a separate PR?

Reply:
:+1: https://github.com/lowRISC/opentitan/pull/27284